CMakeLists.txt

```diff
@@ -1697,7 +1697,7 @@ if(KEYFINDER)
     file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/download")
     ExternalProject_Add(libKeyFinder
```

Comment:
here we should also adjust the capitals.

Reply:
Done.